<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H_probe_test</class>
 <widget class="QWidget" name="H_probe_test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>开关稳定时间</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_dec_sleep_time"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>状态切换时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_switch_time"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>保存路劲</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_save_path"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_select_path">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>采集次数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_collect_time"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>单次间隔时间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit__wait_time"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox_stop">
      <item>
       <property name="text">
        <string>Hz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kHz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GHz</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_stopfreq"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_power"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_startfreq"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_start">
      <item>
       <property name="text">
        <string>Hz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kHz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GHz</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_freq_iter"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>开始频率：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>结束频率：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>中频带宽：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>dBm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>频点数：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBox_if">
      <item>
       <property name="text">
        <string>Hz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kHz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GHz</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>功率：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_if"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_vna_reset">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>矢网重置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonapply">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>202</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>Trace</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_S11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>S11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_S12">
      <property name="text">
       <string>S12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_S21">
      <property name="text">
       <string>S21</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_S22">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>S22</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manual">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_interal">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>Interal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update_trace">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_elec_switch">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开关切换</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_ele_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_ele_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_ele_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="checkBox_ele_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_ele_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_ele_6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_ele_7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBox_ele_8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_ele_9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_ele_10">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox_ele_11">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="checkBox_ele_12">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_ele_13">
      <property name="text">
       <string>13</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox_ele_14">
      <property name="text">
       <string>14</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_ele_15">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="checkBox_ele_16">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
